src/rez/tests/test_plugin_manager.py

```diff
@@ -68,6 +69,13 @@ def test_new_loading_style(self):
                 "package_repository", "cloud")
             self.assertEqual(cloud_cls.name(), "cloud")
 
+    def test_load_plugin_from_entry_points(self):
+        """Test loading rez plugin from setuptools entry points"""
+        with restore_pip("baz", os.path.join(self.data_path("extensions"), "baz")):
```

Comment:
It would be safer to either just install into a temporary directory, or just commit the dist-info folder in the repo to avoid having to install it.